src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -889,11 +897,27 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
     retry_exception = Exception(None)
     for _ in range(0, max_retry):
         try:
-            return sdk_no_wait(no_wait, client.create_or_update,
-                               resource_group_name=resource_group_name,
-                               resource_name=name,
-                               parameters=mc,
-                               custom_headers=headers)
+            print('AKS cluster is creating, please wait...')
```

Comment:
I suggest use logger. Usually the output of create command is properties of the created resource in json format. Other info can be output to stderr.

Reply:
fixed, thanks

Reply:
fixed, thanks